src/protocolsupport/protocol/typeremapper/sound/SoundRemapper.java

```diff
@@ -124,6 +124,7 @@
 			registerRemapEntry("entity.hostile.death", "damage.hit", ProtocolVersionsHelper.BEFORE_1_7);
 			registerRemapEntry("entity.generic.hurt", "damage.hit", ProtocolVersionsHelper.BEFORE_1_7);
 			registerRemapEntry("entity.player.death", "damage.hit", ProtocolVersionsHelper.BEFORE_1_7);
+			registerRemapEntry("entity.player.hurt", "damage.hit", ProtocolVersionsHelper.BEFORE_1_6);
```

Comment:
Are you sure 1.7 doesnt need this remap too? From the remap above it seems like it.